tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -124,14 +130,15 @@ def expired_keys(self):
     def svc(self, db_session, aes_service):
         return RSAKeyService(db_session, aes_service)
 
-    @pytest.fixture
-    def jwk(self, patch):
-        return patch("lms.services.rsa_key.jwk")
-
     @pytest.fixture
     def rsa(self, patch):
         return patch("lms.services.rsa_key.rsa")
 
+    @pytest.fixture
+    def rsa_key(self):
+        key_data = b"-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEA2BNk2+UPDI3ihR8MXMi11xNzaaS0NiVDTzA7MczlRh9WAqRW\n3onaCmNgFXmq+hHy1HKIPohMn/N9TvmtrvhTQV/skn4lADYyfxFTsKZkFPJWoOx3\nsbRJPZVx7HiPaHb72zpTfhwsg5HHa3Sn5Qs8Fybyo33sjbTX1sGZUCq3JGbtEgOi\ni2j19ICS+87AXcMFmCGkaTCFwDoQ38swYDU1YjwhINxx+N5hU19vxA8ZmNexnrpK\nhvF9C2ejS63xJxMCInfJZFb+Yh2zrzExMLQLpTUEBd0tw1ng66xqkB2LUwr0j+Zd\no5/swEepQuOtuLq9omFP345kYIk6IIl0NMcnpQIDAQABAoIBAAWMdyXVcg3WU+rF\nIBeYeODy/F79YYciOe8ETYGku364PcikfAcyPr6CP3yo5FZ2uqFj51jeit2XQcPG\nK7fHSgRvAex5UrQFcEXTX4goOBnSM+rLsvKSu/tZ0kEQ3H/oGrIa9pUE6xi3pOMY\nlECQiaSMXCVAjdNN9LZfwAyvHocpYf8GFo/NBmf1vheDReFiO9Rpx9QzYOO8G6i7\n26aBIiQOHHd+utXF6UXaOW+CHT0fTUWLbUVeaCIWJWdcFzkTcaVGBdEj+H8wfjb3\nW6xwZ/0b7Nma4z+OB1nQ9mCc6+OVoPRylpjP8KAkqGg3sD0TXdolSxdriKGGCKNv\nBqf0iSECgYEA7mIw1hS0hElhlAZqdMFEc761Kect/8btGOZ8vEoVVea4d7a+NJuZ\n8NKRZ7ROcYZjirE3QXAVgGQZTP9S4XuZITVF5DFsR1jjHSKjO/X8LTjo4WzTH1ol\nAJW6zEC8Wpx8QRwfjPQi5IJfnLf1cAJiLuQzrk+XRNvUUa+Am+cEu0UCgYEA6Ast\ng5R7sHSqHnXlMppPNvJQy4bK70+G2VEJF4nY+yVo8n8l/D5zG83X4feepuUvoiAV\nE9/1DCHUSj/9XJlPpgaA/QSVuSoCzLkVFv3QnwMV82hRr6sBw+QlO0mnGfyE/Xl1\niLKSGhCEqGMXuqvyunViheX/QOIpxnTnVffFUOECgYEAtM+bdTXdOh2iurEnHAnf\nzt8O8Iqd9EzBV8qzP+n9RtuqfXsJyENhvy0oBv3XJfqmE/OZErReSrUiD6n2BntG\nSc7rhUsLcw5zrYyxPXC15uMsmJd/h19Lg6cOOZC8jQn2oTggojwnHyXYObm6m0vj\nhuemX4eVGDCZWaABr77JcvUCgYAWjree+flIPx8mlOlyEOQwgD/weSsTNpTyXVlX\n63OnfoyEPm4P5nZENq+M7QiESvVlel7yLqxgwI0lprDXpqPCjRFPB3oSpQ3enwN4\n17XHL4KbxgFi5WnnhC9GYzOWaCD5jywo3MstM3vh7rgo0nxnOfAY+jHlOdc7zrOK\nfOq2oQKBgQDtSLLelV/u24iHfsOVMO5Wpdytdkls3uG410vlgMteytn682vg4knT\n+0x4M3CnQoZ6l9MpuEFB/a9Ya6kZsL3n3duQwMcOYj9YF5Oj6xzlVTGdTM/+1SFK\nAHCI/NHS3Ap2RljTSiMKpN3eRAdp3KoOJOvMCIHl6MMoRp2bVPgplw==\n-----END RSA PRIVATE KEY-----\n"
```

Comment:
I wonder if GitHub's secret scanning might get tripped up by an apparent secret like this in repositories, or whether it is smart enough to realize that it is probably not an issue in `*_test.*` files.

Reply:
I haven't seen a complaint about being pushed it in this branch so I assume it being clever about the test part.